ts/industries.ts

```diff
@@ -395,12 +395,82 @@ export const industrySvgPaths: Partial<Record<IndustryName, Record<string, PathA
     'watertower_kanaskat_style3': waterTower,
     'watertower_kanaskat_style4': waterTower,
     'watertower_small': waterTower,
-    'WaterWell': waterTower,
     'engineshed_style1': largeEngineHouse,
     'engineshed_style2': largeEngineHouse,
     'engineshed_style3': largeEngineHouse,
     'engineshed_style4': largeEngineHouse,
     'Woodrick': {
         'building': rect(-130, -65, 260, 130),
     },
+    'WaterWell': {
```

Comment:
this is only the farm industry chain, but i am ready to trace the gold industry chain as well